src/main/jbake/content/docs/components/metrics/index.adoc

```diff
@@ -5,7 +5,25 @@ Thomas Segismont
 :jbake-type: page
 :jbake-status: published
 
-* link:installation.html[Installation]
-* link:configuration.html[Configuration]
-* link:grafana_integration.html[Grafana integration]
+Hawkular Metrics is the metric data store part of the http://www.hawkular.org/[Hawkular] organization.
+It is currently used a the metric storage engine for Hawkular but it is also be independently. The project relies on https://cassandra.apache.org/[Apache Cassandra] as a backend.
```

Comment:
My english is not very good, but this phrase:
"It is currently used a the metric storage engine for Hawkular but it is also be independently."
what about
<< It is currently used as the metric storage engine for Hawkular but it can also be used independently. >>